### ISSUE TITLE
Minor weirdness with agenda popup

### ISSUE THREAD

**Issue by cabo** [ietf-tools/datatracker#7223]
### Describe the issue

<img width="978" alt="Screenshot 2024-03-18 at 14 20 13" src="https://github.com/ietf-tools/datatracker/assets/23118/3b7fbc05-2dc2-4d93-a26e-b241485004cb">

(Weird Email Address links, after clicking on a document link in an agenda)

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by jennifer-richards** [comment]
Odd - and not seeing it here. What browser?

**Comment by rjsparks** [comment]
I can reproduce in Chrome Version 122.0.6261.129 (Official Build) (arm64)

**Comment by cabo** [comment]
Yes, Chrome Version 122.0.6261.112 (Official Build) (x86_64) for me.
Works fine in Safari Version 15.6.1 (15613.3.9.1.16, 15613)
